src/pages/utils.tsx

```diff
@@ -131,5 +131,5 @@ export function isValidAccountAddress(accountAddr: string): boolean {
   // account address is 0x{64 hex characters}
   // with multiple options - 0X, 0x001, 0x1, 0x01
   // can start with that and see if any fails to parsing
-  return /^(0[xX])?[a-fA-F0-9]{1,64}$/.test(accountAddr);
+  return /^(0[xX])?[a-fA-F0-9]{64}$/.test(accountAddr);
```

Comment:
I believe leading zeros are truncated from addresses, so the previous regex was correct. https://aptos.dev/guides/guide-for-system-integrators/#account-identifiers